.kitchen.yml

```diff
@@ -27,9 +27,13 @@ platforms:
     run_list:
       - recipe[apt::default]
   - name: windows-2012r2
+    driver_config:
```

Comment:
Could you explain this change? Also, why do we need a separate test suite (vs. using the default one for Windows)?